LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 gatsbyjs
```

Comment:
Wrong copyright

Reply:
Will remove and just add the standard Github one.